src/main/scala/ai/privado/exporter/CollectionExporter.scala

```diff
@@ -77,13 +78,38 @@ class CollectionExporter(cpg: Cpg) {
         })
     })
 
+    collectionMethods.foreach(collectionMethod => {
+      collectionMethod.local
+        .and(_.tag.nameExact(Constants.id))
+        .foreach(localVar => {
+          try {
+            def addToMap(parameterId: String): Unit = {
```

Comment:
rename the variable `parameterId` to `localId` or `localVariableId`

Reply:
Changes done here: https://github.com/Privado-Inc/privado-core/pull/370/commits/a2b4c1f93171c273bb23686474e8db69c4ae3235

Reply:
@hiteshbedre Looks like you updated the wrong function, update it on line number 86 function

Reply:
Corrected here: https://github.com/Privado-Inc/privado-core/pull/370/commits/62c970a3cf8860ab01497b4cc82b3e7e2e36d41e